<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditWidget</class>
 <widget class="QWidget" name="ImageEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitterMainContent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tableWidgetImageList">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>0</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>126</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>サムネイル</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ファイル名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>パス</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>サイズ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>既存タグ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>既存キャプション</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="widget_PreviewArea_2">
      <layout class="QVBoxLayout" name="verticalImagePreview">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="ImagePreviewWidget" name="ImagePreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>0</height>
          </size>
         </property>
         <widget class="QLabel" name="labelPreviewTitle">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>177</width>
            <height>21</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>プレビュー</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutControlArea">
     <item>
      <widget class="QGroupBox" name="groupBoxEditOptions">
       <property name="title">
        <string>編集オプション</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_EditOptions_2">
        <item>
         <widget class="QLabel" name="labelResizeOption">
          <property name="text">
           <string>リサイズ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxResizeOption">
          <item>
           <property name="text">
            <string>512x512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>768x768</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024x1024</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUpscaler">
          <property name="text">
           <string>アップスケーラー</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxUpscaler">
          <property name="currentText">
           <string notr="true">None</string>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStartProcess">
       <property name="text">
        <string>処理開始</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImagePreviewWidget</class>
   <extends>QWidget</extends>
   <header location="global">ImagePreviewWidget</header>
   <container>1</container>
   <slots>
    <slot>load_image()</slot>
    <slot>update_model_options()</slot>
    <slot>send_vision_prompt()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
